pkgs/top-level/all-packages.nix

```diff
@@ -1688,7 +1688,13 @@ in
 
   b3sum = callPackage ../tools/security/b3sum {};
 
-  backblaze-b2 = callPackage ../development/tools/backblaze-b2 { };
+  backblaze-b2 = callPackage ../development/tools/backblaze-b2 {
+    python3Packages = python3Packages.override {
+      overrides = self: super: {
+        arrow = python3Packages.callPackage ../development/python-modules/arrow/2.nix { };
```

Comment:
`self.callPackage` is the correct `callPackage` to use. In this case it won't matter because you change only one attribute, but still, we should avoid bad examples.

Please perform the override inside the expression of `backblaze-b2` to reduce bloat in `all-packages.nix`.

Reply:
Thanks to pointing this out. I'm still learning the correct approach .